.editorconfig

```diff
@@ -0,0 +1,3 @@
+[pom.xml]
+indent_style = space
+indent_size = 2
```

Comment:
I find XML hard enough to parse without having such a small indentation difference.  Can we leave out the indent space?  I have a feeling it'll just cause debate (see: this very comment :P)

Reply:
4 spaces? I don't care either way, I just want it to be _consistent_.